### ISSUE TITLE
Support heredocs in the validator for `COPY` instructions

### ISSUE THREAD

**Issue by rcjsuen** [rcjsuen/dockerfile-utils#112]
```Dockerfile
# syntax=docker/dockerfile:1
FROM alpine
COPY <<-EOT /test.txt

	hello
EOT
```
```
$ DOCKER_BUILDKIT=1 docker build --no-cache .
[+] Building 1.4s (11/11) FINISHED                                                                                                                  
 => [internal] load build definition from Dockerfile                                                                                           0.0s
 => => transferring dockerfile: 111B                                                                                                           0.0s
 => [internal] load .dockerignore                                                                                                              0.0s
 => => transferring context: 2B                                                                                                                0.0s
 => resolve image config for docker.io/docker/dockerfile:1                                                                                     0.2s
 => CACHED docker-image://docker.io/docker/dockerfile:1@sha256:443aab4ca21183e069e7d8b2dc68006594f40bddf1b15bbd83f5137bd93e80e2                0.0s
 => [internal] load .dockerignore                                                                                                              0.0s
 => [internal] load build definition from Dockerfile                                                                                           0.0s
 => [internal] load metadata for docker.io/library/alpine:latest                                                                               0.1s
 => [internal] preparing inline document                                                                                                       0.0s
 => CACHED [1/2] FROM docker.io/library/alpine@sha256:7580ece7963bfa863801466c0a488f11c86f85d9988051a9f9c68cb27f6b7872                         0.0s
 => [2/2] COPY <<-EOT /test.txt                                                                                                                0.1s
 => exporting to image                                                                                                                         0.0s
 => => exporting layers                                                                                                                        0.0s
 => => writing image sha256:343ade6b67057dd6a97753767c3469acbb99d6cbe9ce56df330d31813fb5f82d                                                   0.0s
```
```JavaScript
const lib = require("dockerfile-utils");
const content = `# syntax=docker/dockerfile:1
FROM alpine
COPY <<-EOT /test.txt

	hello
EOT`;
const edits = lib.validate(content);
console.log(edits.length);
```
```
1
```
Heredocs are not currently supported for `COPY` instructions so we end up flagging empty continuation lines in them. This is the `COPY` version of #107.

**Comment by ohmer** [comment]
Got the same issue. Is there a known workaround?

How can I help to get this done? Not fluent in JS/TS unfortunately...

**Comment by rcjsuen** [comment]
> How can I help to get this done?

@ohmer This has been resolved now. I have released 0.11.0 which will allow the validator to ignore these empty continuation lines in `COPY` instructions.

**Comment by zffocussss** [comment]
> > How can I help to get this done?
> 
> @ohmer This has been resolved now. I have released 0.11.0 which will allow the validator to ignore these empty continuation lines in `COPY` instructions.

when will be published now?very preciate

**Comment by rcjsuen** [comment]
> when will be published now?very preciate

0.11.0 was released in August of 2022 and I have just released a new version of `dockerfile-language-server-nodejs` today with this fix also if that was what you were waiting for. Apologies for the delay and I hope you have a great Sunday.
